### ISSUE TITLE
D3D9: Cannot create texture - Missing textures in Guild Wars 2

### ISSUE THREAD

**Issue by etienne-k** [doitsujin/dxvk#1755]
Since certain changes have been made to the game (uses D3D9) by the game developer, various textures don't load anymore.

I believe this is the culprit:
```
err:   D3D9: Cannot create texture:
  Type:    3
  Format:  INTZ
  Extent:  80x80x1
  Samples: 0
  Layers:  1
  Levels:  1
  Usage:   1
  Pool:    0
```

Could it be that these "INTZ" textures are not implemented? AMD has some [info](https://developer.amd.com/wordpress/media/2012/10/Advanced-DX9-Capabilities-for-ATI-Radeon-Cards_v2.pdf) on that texture format.

### Software information
Guild Wars 2, the game uses D3D9.
I've tried with and without ESync, and with and without DLL overrides. The issue does not occur in plain when starting without DXVK and therefore translating calls to OpenGL ones.
I use WINE_LARGE_ADDRESS_AWARE=1.

In-game screenshot (in the bottom-right corner, you would normally see a minimap): https://en-forum.guildwars2.com/discussion/111978/black-minimap-unable-to-see-red-dots-wvw-pings-or-others

### System information
- GPU: AMD Radeon 5700XT 8GB
- Driver: MESA driver 20.1.0 on elementary 5.1
- Wine version: ge-protonified-nofshack-4.9-x86_64 or tkg-protonified-3.21-x86_64
- DXVK version: 1.2.1

- Core i7 4790K
- 16 GB DDR3
- AMD Radeon 5700XT 8GB

### Apitrace file
[gw2.trace.7z](https://mega.nz/file/3TwHERDQ#clKy5RzhxQBHMCf7aPYCJWPkV_wMm87e1Czta410Y74)

### Log file
[gw2.log](https://github.com/doitsujin/dxvk/files/5172974/gw2.log) (the log applying to the actual in-game part starts at line 139; before that, the log output was produced while the game launcher was active)

**Comment by doitsujin** [comment]
We do support INTZ, but the game for some reason specifies render target usage instead of depth-stencil which seems like an app bug?

**Comment by Iglu47** [comment]
game: gw2-64.exe build:105843
this is a 64-bit client, so no need to use `WINE_LARGE_ADDRESS_AWARE=1`

my spec
- core i7 4790K / 16 gb ddr3
- nvidia proprietary driver: 450.56.6; gtx nvidia 1080ti
- arch linux, kernel: 5.7.19-tkg-bmq
- wine-staging-5.9.r11.g4e692b53
- dxvk 1.7.1-6 9433d0cbf33e17b097041ecfb52f9f0cf31276fd

[Gw2-64_d3d9.log](https://github.com/doitsujin/dxvk/files/5173996/Gw2-64_d3d9.log)

<details><summary>In-game screenshot</summary>
<p>

![gw2-64-105843](https://user-images.githubusercontent.com/25866099/92226007-b82d8d00-eeac-11ea-8259-661d34b9b00a.png)

</p>
</details>

> Guild Wars 2, the game uses D3D9.
> * DXVK version: 1.2.1

i thought d9vk was only included in dxvk 1.5. how is this possible? https://github.com/doitsujin/dxvk/releases/tag/v1.5

**Comment by jrugia** [comment]
@etienne-k Can you please try the game with more recent version of DXVK?

Version 1.2.1 has been released over a year ago (19th May 2019) and things have changed quite a bit since then, especially for DX9.

**Comment by doitsujin** [comment]
Good point actually, my brain somehow read that as 1.7.1 instead.

**Comment by etienne-k** [comment]
Thanks for all the replies! Before cloning/building/installing DXVK from master, I went over to Lutris and saw the "DXVK version" option, a bit hidden in the advanced runner properties. There, an old version was selected (I just saw the hash, and didn't take note of it; now it disappeared from the combobox). I selected "1.7.1L-743f309" instead and started the game. Now the issue is gone!
Because the only thing that changed was the game, I didn't think the problem would be solved by a newer DXVK version already; I should've had more faith in this project. :)

Thanks again for your help.
